components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -374,6 +374,38 @@ type Deployment {
     boundServiceInstanceNames: [String!]
 }
 
+type Pod {
+    name: String!
+    nodeName: String!
+    namespace: String!
+    restartCount: Int!
+    creationTimestamp: Timestamp!
+    labels: Labels!
+    status: PodStatusType!
+    containerStates: [ContainerState!]!
+    json: JSON!
+}
+
+enum PodStatusType {
+    PENDING
+    RUNNING
+    SUCCEEDED
+    FAILED
+    UNKNOWN
+}
+
+type ContainerState {
+    state: ContainerStateType!
+    reason: String
```

Comment:
Return empty `string` if `reason` or `message` is not provided. Kubernetes API also returns empty `string`

```
    reason: String!
    message: String!
```